__init__.py

```diff
@@ -197,6 +197,28 @@ def _handle_switch(self, message):
         self.log.debug("Entity: %s" % entity)
         self.log.debug("Action: %s" % action)
 
+        # Handle turn on/off all intent
+        try:
+            if self.voc_match(entity,"all_lights"):
+                domain = "light"
+            elif self.voc_match(entity,"all_switches"):
+                domain = "switch"
+            else:
+                domain = None
+
+            if not domain is None:
```

Comment:
`if domain is not None` is the preferred way to write this

```
>>> foo = None
>>> if foo is not None:
...   print('fi')
... 
>>> foo = 'hi'
>>> if foo is not None:
...   print('bye')
... 
bye
```

it is more readable

Reply:
OK